### PR TITLE
[Feat] Search and Filter Page Background Image

### PR DESCRIPTION
**Pull request by jessiiepeng**
## Notion ticket link

[Search and Filter BG Image](https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=aedc8d99cd8b437cb556ece10f3c0e3c&p=130533dc9d8a462b824114f5d761b348&pm=c)

## Implementation description
* added background image 
* background image doesn't show in mobile view

## Steps to test
1. Go to http://localhost:3000/magazine/search_results/ and verify that background image works as expected.
2. In file `mockReviews.ts` you can add / delete reviews and check that the background image size works as expected.
3. Go to mobile view and verify that bg image isn't there.

Demo:
https://drive.google.com/file/d/15bGR_BblDI5F_rnQEc_CjjSMWQNytKyw/view?usp=sharing
Screen Shot 2022-08-03 at 6 42 32 PM

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have run docker-compose up and my project compiled
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by susheelk — COMMENTED** [review]
Code looks good! I'd like to wait on a design review though

**Comment by jessiiepeng** [comment]
> Code looks good! I'd like to wait on a design review though

I got the approval from Amanda :D / also made it into 5 books instead of 4.

**Review by susheelk — APPROVED** [review]
LGTM!

**Pull request merged**
